pages/blog/_slug.vue

```diff
@@ -111,7 +123,11 @@ export default {
   async asyncData({ $content, params }) {
     const article = await $content('blog', params.slug).fetch()
 
-    return { article }
+    let relatedArticles = await $content('blog').fetch()
+    relatedArticles = relatedArticles.filter(relatedArticle => article.slug !== relatedArticle.slug)
+    const randIndex = Math.floor(Math.random() * (relatedArticles.length - 2))
```

Comment:
Reason for doing `relatedArticles.length - 2`

Reply:
yes relatedArticles should always return 2 articles so largest randIndex will be 1 less than last index. while slicing 2 articles it'll include the last article also